### ISSUE TITLE
Improve contrast ratio of 'Change Amount' link

### ISSUE THREAD

**Issue by youriwims** [mozilla/donate-wagtail#319]
### Issue: [Elements must have sufficient color contrast](https://dequeuniversity.com/rules/axe/3.3/color-contrast?application=msftAI)

**Element path:** `.donation-update__toggle`

**Snippet(s):** `<a href="#donate__amount-edit" class="donation-update__toggle" data-amount-toggle="">Change amount</a>`

**How to fix:**
Fix any of the following:
- Element has insufficient color contrast of 4.41 (foreground color: #595cf3, background color: #f2f2f2, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1

====

More Info:

Comments: n/a
Resources: n/a

**Comment by nicklee** [comment]
Changed to mozilla's dark blue #0d10bf

**Comment by youriwims** [comment]
Looks good.
